RELEASE_NOTES.rst

```diff
@@ -21,6 +21,106 @@
+- Fix Airflow serialization for ``namedtuple`` (#37168)
+- Fix bug with clicking url-unsafe tags (#37395)
+- ``Treeview`` - deterministic and new getter (#37162)
+- Fix permissions of parent folders for log file handler (#37310)
+- Fix permission check on DAGs when ``access_entity`` is specified (#37290)
+- Fix the value of ``dateTimeAttrFormat`` constant (#37285)
+- Resolve handler close race condition at triggerer shutdown (#37206)
+- Fixing status icon alignment for various views (#36804)
+- Remove superfluous ``@Sentry.enrich_errors``  (#37002)
+- Use execution_date= param as a backup to base date for grid view (#37018)
+- Handle SystemExit raised in the task. (#36986)
+- Revoking audit_log permission from all users except admin (#37501)
+- Fix broken regex for allowed_deserialization_classes (#36147)
+- Fix the bug that affected the DAG end date. (#36144)
+- Adjust node width based on task name length (#37254)
+- fix: PythonVirtualenvOperator crashes if any python_callable function is defined in the same source as DAG (#37165)
+- Fix collapsed grid width, line up selected bar with gantt (#37205)
+- Adjust graph node layout (#37207)
+- Revert the sequence of initializing configuration defaults (#37155)
+- Displaying "actual" try number in TaskInstance view (#34635)
+- Bugfix Triggering DAG with parameters is mandatory when show_trigger_form_if_no_params is enabled (#37063)
+- Secret masker ignores passwords with special chars (#36692)
+- Fix DagRuns with UPSTREAM_FAILED tasks get stuck in the backfill. (#36954)
+- Disable ``dryrun`` auto-fetch (#36941)
+- Fix copy button on a DAG run's config (#36855)
+- Fix bug introduced by replacing spaces by + in run_id (#36877)
+- Fix webserver always redirecting to home page if user was not logged in (#36833)
+- REST API set description on POST to ``/variables`` endpoint (#36820)
+- Sanitize the conn_id to disallow potential script execution (#32867)
+- Fix task id copy button copying wrong id (#34904)
+- Fix security manager inheritance in fab provider (#36538)
+- Avoid ``pendulum.from_timestamp`` usage (#37160)
+
+Miscellaneous
+"""""""""""""
+- Bump ``undici`` from ``5.26.3`` to ``5.28.3`` in ``/airflow/www`` (#37493)
+- Add Python ``3.12`` exclusions in ``providers/pyproject.toml`` (#37404)
+- Remove ``markdown`` from core dependencies (#37396)
+- Remove unused ``pageSize`` method. (#37319)
+- Add more-itertools as dependency of common-sql (#37359)
+- Replace other ``Python 3.11`` and ``3.12`` deprecations (#37478)
+- Include ``airflow_pre_installed_providers.txt`` into ``sdist`` distribution (#37388)
+- Turn Pydantic into an optional dependency (#37320)
+- Limit ``universal-pathlib to < 0.2.0`` (#37311)
+- Allow running airflow against sqlite in-memory DB for tests (#37144)
+- Add description to ``queue_when`` (#36997)
+- Updated ``config.yml`` for environment variable ``sql_alchemy_connect_args``  (#36526)
+- Bump ``aiohttp`` min version to avoid ``CVE-2024-23829`` and ``CVE-2024-23334`` (#37110)
```

Comment:
This can be skipped from change log it's not relevant for airflow-core change log